crates/pop-cli/src/commands/new/parachain.rs

```diff
@@ -35,9 +38,9 @@ pub struct NewParachainCommand {
 		short = 't',
 		long,
 		help = "Template to use.",
-		value_parser = crate::enum_variants!(Template)
+		value_parser = crate::enum_variants!(ParachainTemplate)
```

Comment:
This enum_variants! macro should be moved to pop_common. Expect it still exists in both contracts and parachains crates.